x/sequencer/types/codec.go

```diff
@@ -38,3 +41,11 @@ var (
 	InterfaceReg = cdctypes.NewInterfaceRegistry()
 	ModuleCdc    = codec.NewProtoCodec(InterfaceReg)
```

Comment:
This global var supposed to be `AminoCodec`
but currently it breaks this snippet
```
func PubKey(pk *codectypes.Any) (cryptotypes.PubKey, error) {
	cdc := ModuleCdc
	var pubKey cryptotypes.PubKey
	err := cdc.UnpackAny(pk, &pubKey)
	return pubKey, err
}
```
which for some reason uses the global var

Reply:
fixed on last commit